src/test/java/org/foundation/coding/customer/CustomerTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Refactor Code Tasks.
+ */
+package org.foundation.coding.customer;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map.Entry;
+import org.foundation.coding.movie.Movie;
+import org.foundation.coding.movie.NewReleaseMovie;
+import org.foundation.coding.movie.RegularMovie;
+import org.foundation.coding.rental.Rental;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test to customer.
+ */
+public class CustomerTest {
+
+  private Customer customer;
+  private Movie movie;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movie" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)